### PR TITLE
Fix/issue 3542

### PR DESCRIPTION
**Pull request by sivasu37**

### CODE REVIEW

**Comment by MaxJPRey** [comment]
@sivasu37 I fixed the merging conflicts.

File: pyaedt/modules/Boundary.py [hunk 1]
```diff
@@ -460,6 +462,23 @@ def type(self):
...
+    @property
+    def type(self):
+        """
+        Returns
+        -------
+        str
+            Returns Type of the boundary
+        """
+        if self._app.design_type == "Icepak" and self._type == "Source":
+            return "SourceIcepak"
```
Comment (lorenzovecchietti) [inline pyaedt/modules/Boundary.py:474]: Can you add a line in the tests so that this fix get tested? Thanks :)

File: pyaedt/modules/Boundary.py [hunk 2]
```diff
@@ -460,6 +462,23 @@ def type(self):
...
+        else:
+            return self._type
+
+    @type.setter
+    def type(self, type_in):
```
Comment (lorenzovecchietti) [inline pyaedt/modules/Boundary.py:479]: Doctest won't pass unless you write a docstring here

File: pyaedt/modules/Boundary.py [hunk 3]
```diff
@@ -460,6 +462,23 @@ def type(self):
     def type(self, value):
         self._type = value
 
+    @property
+    def type(self):
+        """
+        Returns
+        -------
+        str
+            Returns Type of the boundary
```
Comment (PipKat) [inline pyaedt/modules/Boundary.py:471]:
```suggestion
        """
        Type of the boundary.
        
        Returns
        -------
        str
           Type of the boundary.
```

File: pyaedt/modules/Boundary.py [hunk 4]
```diff
@@ -460,6 +462,23 @@ def type(self):
...
+        else:
+            return self._type
+
+    @type.setter
+    def type(self, type_in):
+        self._type = type_in
+
     @property
     def name(self):
         """Boundary Name."""
```
Comment (PipKat) [inline pyaedt/modules/Boundary.py:484]:
```suggestion
        """Boundary name."""
```

**Review by maxcapodi78 — COMMENTED** [review]
It looks to me that the type props and setter is declared twice in the same class. Please review and merge eventually the content

**Comment by sivasu37** [comment]
@maxcapodi78 ,  Sorry, Yes I noticed the duplication. I will fix and comment on the PR later. Then we can merge.

**Comment by sivasu37** [comment]
I will create a new PR for the changes

**Pull request closed without merging**